app/controllers/courses/courses_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Courses::CoursesController < Courses::Controllers
+  load_and_authorize_resource
+
+  def show #:nodoc:
+  end
+
+  def new #:nodoc:
+  end
+
+  def create #:nodoc:
+    @course.course_users.build(user: current_user, name: 'name', role: :owner)
+
+    if @course.save
+      redirect_to edit_course_path(@course), notice: t('.notice', title: @course.title)
+    else
+      render 'new'
+    end
+  end
+
+  def edit #:nodoc:
+  end
+
+  def update #:nodoc:
+    if @course.update_attributes(course_params)
+      redirect_to course_path(@course), notice: t('.notice', title: @course.title)
+    else
+      render 'edit'
+    end
+  end
+
+  def destroy #:nodoc:
+  end
+
+  private
+
+  def course_params
```

Comment:
do we need to doc this also ?  function <code>model_name_params </code> are supposed to be put in every controller.